transport/cluster.go

```diff
@@ -238,6 +306,59 @@ func cqlRowIntoNode(r frame.Row) (*Node, error) {
 	}, nil
 }
 
+func cqlRowIntoStrategy(r frame.Row) (*strategy, error) {
+	strat, err := r[replication].AsStringMap()
+	if err != nil {
+		return nil, fmt.Errorf("strat and rf column: %w", err)
+	}
+
+	s, ok := strat["class"]
+	if !ok {
+		return nil, fmt.Errorf("strategy map should have a 'class' field")
+	}
+	delete(strat, "class")
+
+	switch {
+	case s == "org.apache.cassandra.locator.SimpleStrategy" || s == "SimpleStrategy":
+		rfs, ok := strat["replication_factor"]
+		if !ok {
+			return nil, fmt.Errorf("replication_factor field not found")
+		}
+		rf, err := strconv.ParseUint(rfs, 10, 32)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse replication factor as unsigned int")
+		}
+		return &strategy{
+			strat: simple,
+			rf:    repFactor(rf),
+		}, nil
+	case s == "org.apache.cassandra.locator.NetworkTopologyStrategy" || s == "NetworkTopologyStrategy":
+		dcRFs := make(DcRFsMap, len(strat))
+		for k, v := range strat {
+			rf, err := strconv.ParseUint(v, 10, 32)
+			if err != nil {
+				return nil, fmt.Errorf("could not parse replication factor as int")
+			}
+			dcRFs[k] = repFactor(rf)
+		}
+		return &strategy{
+			strat: networkTopology,
+			dcRF:  dcRFs,
+		}, nil
+	case s == "org.apache.cassandra.locator.LocalStrategy" || s == "LocalStrategy":
+		return &strategy{
+			strat: local,
+			rf:    1,
+		}, nil
+	default:
```

Comment:
Since we aren't supporting custom strategies, maybe we should return error, when unknown strategy appears (just for now)?

Reply:
![Screenshot from 2022-03-29 18-29-23](https://user-images.githubusercontent.com/29147083/160660666-dfaf0018-7868-4c37-b38f-9bb3a4f3976b.png)

That's keyspace query returned from my scylla when 'make scylla-up' is called, the last row already contains unknown by us strategy, we could ignore it for now? But having additional information won't hurt us either.